test_helper.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, print_function, unicode_literals
+
+from tempfile import mkdtemp
+
+
+def gettext(s): return s
+
+HELPER_SETTINGS = dict(
+    INSTALLED_APPS=[
+    ],
+    LANGUAGE_CODE='en',
+    LANGUAGES=(
+        ('en', gettext('English')),
+        ('fr', gettext('French')),
+        ('it', gettext('Italiano')),
+    ),
+    FILE_UPLOAD_TEMP_DIR=mkdtemp(),
+    SITE_ID=1,
+)
+
+
+def run():
+    from djangocms_helper import runner
```

Comment:
hmm, I don't think we should depend on 3rd party (despite it being a package maintained by you!) for running simple Django tests, it seems like using `settings.configure` like https://github.com/carljm/django-secure/blob/master/runtests.py would suffice for this